src/tensorflow.cc

```diff
@@ -2031,10 +2031,10 @@ ModelInstanceState::ProcessRequests(
       // For certain input data, sometimes a model can return an empty
       // tensor as output. If so, then we must attach an empty output
       // to the response.
-      if (TRITONTF_TensorData(output_tensor) == nullptr) {
+      if (TRITONTF_TensorDataByteSize(output_tensor) == 0) {
         std::vector<int64_t> empty_shape{0};
         responder.ProcessTensor(
-            name, datatype, empty_shape, nullptr,
+            name, datatype, empty_shape, TRITONTF_TensorData(output_tensor),
```

Comment:
Is there a functional difference in passing `nullptr` vs. tensor data for an empty tensor here? Would `nullptr` be incorrect?

Reply:
Even though not documented.. Internally the nullptr pointer comes from a `.data()` call on `tensorflow::Tensor` object. I couldn't see the documentation. But atleast the doc of std::vector.data() says that it may or may not be nullptr